cli/src/klio_cli/commands/job/create.py

```diff
@@ -195,16 +193,15 @@ def _parse_python_version(self, python_version):
         if len(python_version) < 2 or len(python_version) > 5:
             raise click.BadParameter(invalid_err_msg)
 
-        # 3.x -> 3x; 3.x.y -> 3x
-        if "." in python_version:
-            python_version = "".join(python_version.split(".")[:2])
+        # keep only the major and minor version information
+        python_version = ".".join(python_version.split(".")[:2])
 
-        if python_version not in VALID_BEAM_PY_VERSIONS_SHORT:
+        if python_version not in VALID_BEAM_PY_VERSIONS:
             raise click.BadParameter(invalid_err_msg)
 
         # Dataflow's 3.5 image is just "python3" while 3.6 and 3.7 are
         # "python36" and "python37"
-        if python_version == "35":
+        if python_version == "3.5":
```

Comment:
Is this still needed?  It looks like beam's image for 3.5 follows the same scheme: https://hub.docker.com/r/apache/beam_python3.5_sdk

Reply:
Ah, good catch! Removed.